dessia_common/workflow/utils.py

```diff
@@ -65,3 +69,44 @@ def nonblock_variables_to_script(nonblock_variables, prefix, imports, imports_as
         imports_as_is.extend(nbv_script.imports_as_is)
         nbvs_str += f"{prefix}variable_{nbv_index} = {nbv_script.declaration}\n"
     return nbvs_str
+
+
+def update_imports(input_type, input_name):
+    """ Update imports based on the schema of the input. """
+    schema = get_schema(annotation=input_type, attribute=SchemaAttribute(input_name))
+    return schema.get_import_names(import_names=[])
+
+
+def generate_input_string(signature: str, sequence):
+    """ Generate input as string. """
+    if len(sequence) > 1:
+        instances = [f"\n\t{o.__class__.__name__}{signature}" for o in sequence]
+        return f"[{','.join(instances)}\n]"
+    return f"{sequence[0].__class__.__name__}{signature}"
+
+
+def is_object_sequence(sequence):
+    """ Checks if a sequence contains serializable objects. """
+    return all(isinstance(element, SerializableObject) for element in sequence)
+
+
+def process_value(value):
+    """ Processes a value based on its content. """
+    if not value:
+        return repr(value)
+
+    if is_object_sequence(value):
+        signature = "('Set your arguments here')"
+    elif is_file_or_file_sequence(value):
+        signature = ".from_file('Set your filepath here')"
+    else:
+        return repr(value)
+
+    return generate_input_string(signature, value)
```

Comment:
This is still not safe for non sequence objects as you evaluate len in `generate_input_string`. Consider : 

```py
    if not value: # Under certain condition (false objects ruled out from is_sequence functions) you could remove this
        return repr(value)

    if is_object_sequence(value):
        return generate_input_string("('Set your arguments here')", value)
    elif is_file_or_file_sequence(value):
        return generate_input_string(".from_file('Set your filepath here')", value)
    return repr(value)
```

If your is_object_sequences and is_file_or_file_sequence functions rule false objects out, you can even remove the first check.